src/v/utils/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ v_cc_library(
     file_io.cc
     base64.cc
     retry_chain_node.cc
+    vint.cc
```

Comment:
would this make sense to add back at like the 2nd commit?

Reply:
I think Michal also brought it up, personally I placed a second batch of commits to vint.h in here because it specifically had to do with implementing parsing flex headers. And the source that necessitated the change was introduced in this commit. Thoughts?